composer.json

```diff
@@ -58,6 +58,8 @@
   },
   "minimum-stability": "dev",
   "require": {
+    "php": ">=7.1",

```

Comment:
We actually support >=7.2, but indeed other extensions are wrongly saying 7.1

Reply:
I've raised it as part of this PR because I'm using nullable parameters. Shall I raise the version to 7.2 instead? :)

Reply:
I think it makes sense https://github.com/oat-sa/tao-community/blob/master/composer.json#L6

Reply:
Done :point_right: e206ec10bac0ad8ca248d95f136e9c27c52cc3e3